### DISCUSSION TITLE
Nvim become very sluggish after update

### DISCUSSION THREAD

**Discussion by muzichen** [LazyVim/LazyVim#5106] [Q&A]
After updating to the latest version, it has become very sluggish and difficult to use. Has anyone else encountered the same situation? The setup is a MacBook Pro M1 Pro.

**Comment by timjackleus** [comment]
Could it be that you have many config files for plugins that are no longer used by LazyVim, but the config is causing them to still be installed and they overlap? I'm on M1 as well and the latest update has been smooth as butter for me.

**Reply by ishanzuaim** [reply]
I have little to no added plugins (just vimux and auto-save).

Which language are you using? I noticed its very slow for just typescript, works fine when I am using golang

**Comment by ishanzuaim** [comment]
Same, running on M1 air and opening modals (fzf) and autocomplete feels very slow

I switched to nvim-cmp and its fast again. Modal opening is still a bit slow but I can tolerate that for now

**Reply by ibanks42** [reply]
Yeah I had the same. Switched back to nvim-cmp and it's much faster. Haven't tried blink with any other languages besides typescript so far though.
